AutoCollection/diagnostic-channel/azure-coretracing.sub.ts

```diff
@@ -37,15 +38,22 @@ export const subscriber = (event: IStandardEvent<Span>) => {
 };
 
 export function enable(enabled: boolean, client: TelemetryClient) {
+    let statsbeat = client.getStatsbeat();
     if (enabled) {
         if (clients.length === 0) {
             channel.subscribe<any>("azure-coretracing", subscriber);
+            if (statsbeat) {
+                statsbeat.addInstrumentation(StatsbeatInstrumentation.AZURE_CORE_TRACING);
```

Comment:
never mind about the other comment about instrumentation.. i think name is a good indicate what each number is for.